1.8/Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM debian:jessie-backports
+
+RUN apt-get update \
+	&& apt-get install -y --no-install-recommends \
+		liblua5.3-0 \
+		libpcre3 \
+		libssl1.0.0 \
+	&& rm -rf /var/lib/apt/lists/*
+
+ENV HAPROXY_MAJOR 1.8
+ENV HAPROXY_VERSION 1.8-rc1
+ENV HAPROXY_MD5 8a9a3fcee37b147c4b5488d5b2fffb3a
+
+# see http://sources.debian.net/src/haproxy/jessie/debian/rules/ for some helpful navigation of the possible "make" arguments
+RUN set -x \
+	\
+	&& buildDeps=' \
+		gcc \
+		libc6-dev \
+		liblua5.3-dev \
+		libpcre3-dev \
+		libssl-dev \
```

Comment:
And add `zlib1g-dev` here.

Reply:
Does HAProxy use this lib for compression?